controller/environment_blackbox_test.go

```diff
@@ -0,0 +1,90 @@
+package controller_test
+
+import (
+	"context"
+	"testing"
+
+	uuid "github.com/satori/go.uuid"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	testsuite "github.com/fabric8-services/fabric8-common/test/suite"
+	"github.com/fabric8-services/fabric8-env/app"
+	"github.com/fabric8-services/fabric8-env/app/test"
+	"github.com/fabric8-services/fabric8-env/configuration"
+	"github.com/fabric8-services/fabric8-env/controller"
+	"github.com/fabric8-services/fabric8-env/gormapp"
+	"github.com/goadesign/goa"
+	"github.com/stretchr/testify/suite"
+)
+
+type EnvironmentControllerSuite struct {
+	testsuite.DBTestSuite
+	service *goa.Service
+	db      *gormapp.GormDB
+	ctrl    *controller.EnvironmentController
+}
+
+func TestEnvironmentController(t *testing.T) {
+	config, err := configuration.New("")
+	require.NoError(t, err)
+	suite.Run(t, &EnvironmentControllerSuite{DBTestSuite: testsuite.NewDBTestSuite(config)})
+	// TODO add test for all different response code
```

Comment:
We should provide all the test for the code inside the PR. TODO for tests should not typically happen. They may be some rear exceptions though. Is there any good reason for that this time? :)